website/docs/concepts/gps_service/README.md

```diff
@@ -0,0 +1,90 @@
+---
+id: gps-overview
+title: GPS
+---
+
+# GPS Service Overview
+
+The GPS service is an integral component of the Amiga's navigation system, utilizing a
+high-precision Ublox module to determine the robot's exact position on Earth.
+This service is indispensable for tasks that require geo-location, from simple
+navigation to complex tasks like field mapping or precision agriculture.
+
+The primary role of the GPS service is to continuously capture and broadcast detailed
+positional data, which other components and services within the Amiga's system can utilize.
+It simplifies the intricacies of satellite-based positioning, offering users straightforward
+access to precise, real-time location data.
+
+In short, the GPS service continuously decodes and transmits location data, allowing users
+and systems to tap into accurate geospatial information without dealing with the
+complexities of GPS data interpretation.
+
+## Standalone Operation
+
+Unlike other services, the GPS service operates independently and is not a client of any other services.
+Its sole responsibility is to interpret and publish the data received from the GPS module.
+
+## Message Types
+
+Clients can subscribe to the GPS service to receive messages, which are typically of type `pvt` or `relposned`.
+These messages contain rich data sets that include the robot's current geospatial coordinates,
+velocity, time, and other relevant GPS data.
+
+:::info INFO
+There are two types of GPS messages: **PVT** and **RELPOSNED**.
+
+**PVT** (Position, Velocity, and Time) messages provide the all-in-one solution: position, velocity,
+and time.
+It contains details like longitude, latitude, altitude, speed, and UTC time.
+
+**RELPOSNED** (Relative Positioning) messages provide relative position information in a
+North, East, Down (N-E-D) frame. It's mainly used for applications requiring relative positioning
+between two receivers, often as a part of Real Time Kinematics (RTK) solutions.
+It shows the difference in position between a "moving" receiver and a "fixed" reference receiver.
+:::
+
+## Requirements
+
+For the GPS service to function optimally, certain conditions must be met:
+
+- The robot must have an active Wi-Fi connection, ensuring real-time data communication.
```

Comment:
Is a wifi connection required for PVT only operation? We should at least specify that pvt can be used without a base station

Reply:
Good point, I wasn't aware of this, but tested it on `boron-banana`.
Addressed this comment on commit bce87e5f

Reply:
Excellent! Thanks for testing